File: C/PRAK503-2210817310009-MarioFrancaWijaya-.c
#include <stdio.h>

int maksimals(int maks, int nilai){
    if(maks > nilai){
        return maks;
    }else{
        return nilai;
    }
}
int minimal(int minim, int nilai){
    if(minim < nilai){
        return minim;
    }else{
        return nilai;
    }
}
int main(){
    int batas = 0;
    int maks = -100000;
    int minim = 100000;
    int bilangan;
    scanf("%d", &bilangan);
    while(batas < bilangan){
        int nilai;
        scanf("%d", &nilai);
        maks = maksimals(maks, nilai);
        minim = minimal(minim, nilai);
        batas++;
    } 
    printf("%d %d", maks, minim);
}

File: Python/PRAK503-2210817310009-Mario Franca Wijaya.py
def maksimal(maks, nilai):
    if maks > nilai:
        return int(maks)
    else:
        return int(nilai)
def minimal(minim, nilai):
    if minim < nilai:
        return int(minim)
    else:
        return int(nilai)

for i in range(3):
    batas = 0
    maks = -100000
    minim = 100000
    bilangan = int(input())
    while batas < bilangan:
        nilai = map(int, input().split())
        for angka in nilai:
            maks = maksimal(maks, angka)
            minim = minimal(minim, angka)
            batas +=1
    print("%d %d"%(maks, minim))